### ISSUE TITLE
Exception when running `pyhanko sign ltvfix`

### ISSUE THREAD

**Issue by tlilja** [MatthiasValvekens/pyHanko#81]
**Describe the bug**
I get an exception when I am trying to run ltvfix on a signed PDF file.

**To Reproduce**
1. Get the CA certificate: 
```wget https://www.sk.ee/upload/files/TEST_of_EE_Certification_Centre_Root_CA.der.crt```
2. Run the pyhanko ltvfix on the document
```pyhanko sign ltvfix --trust TEST_of_EE_Certification_Centre_Root_CA.der.crt  --field Signature1 lorem-ipsum-signed.pdf```

**Expected behavior**
- Run LTV fix successfully

**Environment (please complete the following information):**
```
pyHanko==0.12.0
pyhanko-certvalidator==0.19.4
```

**Additional context**
```Traceback (most recent call last):
  File "$VENV/lib/python3.8/site-packages/pyhanko/sign/validation/dss.py", line 133, in _cms_objects_to_streams
    yield seen[obj_bytes]
KeyError: [...]

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "$VENV/bin/pyhanko", line 8, in <module>
    sys.exit(launch())
  File "$VENV/lib/python3.8/site-packages/pyhanko/__main__.py", line 7, in launch
    cli(prog_name='pyhanko')
  File "$VENV/lib/python3.8/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
  File "$VENV/lib/python3.8/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
  File "$VENV/lib/python3.8/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "$VENV/lib/python3.8/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "$VENV/lib/python3.8/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "$VENV/lib/python3.8/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "$VENV/lib/python3.8/site-packages/click/decorators.py", line 26, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "$VENV/lib/python3.8/site-packages/pyhanko/cli.py", line 632, in ltv_fix
    output = validation.add_validation_info(
  File "$VENV/lib/python3.8/site-packages/pyhanko/sign/validation/__init__.py", line 302, in add_validation_info
    return asyncio.run(coro)
  File "/Users/tlilja/.pyenv/versions/3.8.10/lib/python3.8/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/Users/tlilja/.pyenv/versions/3.8.10/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "$VENV/lib/python3.8/site-packages/pyhanko/sign/validation/dss.py", line 645, in async_add_validation_info
    resulting_dss = DocumentSecurityStore.supply_dss_in_writer(
  File "$VENV/lib/python3.8/site-packages/pyhanko/sign/validation/dss.py", line 443, in supply_dss_in_writer
    dss.register_vri(
  File "$VENV/lib/python3.8/site-packages/pyhanko/sign/validation/dss.py", line 215, in register_vri
    crl_refs = set(
  File "$VENV/lib/python3.8/site-packages/pyhanko/sign/validation/dss.py", line 140, in _cms_objects_to_streams
    dest.append(ref)
AttributeError: 'tuple' object has no attribute 'append'
```

[lorem-ipsum-signed.pdf](https://github.com/MatthiasValvekens/pyHanko/files/8142333/lorem-ipsum-signed.pdf)

**Comment by MatthiasValvekens** [comment]
Yikes, that sure looks like a bug. I'll take a closer look; thanks for the report and the sample file!

**Comment by MatthiasValvekens** [comment]
I've fixed the bug in a bugfix branch off `0.12.0` (see commit message of linked commit). I'll merge that into `master` now. Is this blocking you to the point that you need a bugfix release, or can it wait?

**Comment by tlilja** [comment]
Thanks for the quick fix. While I can probably work around the problem, a bugfix release would be nice.

**Comment by MatthiasValvekens** [comment]
Done, I just released `0.12.1` :).
